File: SecondYear/itsrishti_SrishtiSingh_2226CSE(AI&ML)1107_2/Week_1/pattern6.java
//{ Driver Code Starts
import java.io.*;

class Main {
    // Driver code
    public static void main(String[] args) throws Exception {
        BufferedReader br =
            new BufferedReader(new InputStreamReader(System.in));
        int t = Integer.parseInt(br.readLine().trim());
        while (t-- > 0) {
            int n = Integer.parseInt(br.readLine().trim());
            Solution obj = new Solution();
            obj.printTriangle(n);
        }
    }
}
// } Driver Code Ends


class Solution {

    void printTriangle(int n) {
        
        
        for (int i = 1; i <= n; i++) {
            for (int j = n; j > i; j--) {
                System.out.print(" ");
            }
            for (int s= 1; s <= i; s++) {
                System.out.print("*");
            }
            for (int m = 1; m < i; m++) {
                System.out.print("*");
            }
            System.out.println();
        }
        
    }
}

File: SecondYear/ksh1710_KshitizAgarwal_2226cse1033_2/Week_1/pattern6.cpp
void printTriangle(int n)
{
    int x = 1;
    for (int i = 1; i <= n; i++)
    {
        for (int j = n - 1; j >= i; j--)
        {
            cout << " ";
        }
        for (int k = 1; k <= x; k++)
        {
            cout << "*";
        }
        x += 2;
        cout << endl;
    }
}